curvefs/src/client/fuse_client.h

```diff
@@ -217,6 +226,50 @@ class FuseClient {
     SpaceBaseClient *spaceBase_;
 };
 
+class RenameOperator {
+ public:
+    RenameOperator(uint32_t fsId,
+                   uint64_t parentId,
+                   std::string name,
+                   uint64_t newParentId,
+                   std::string newname,
+                   std::shared_ptr<DentryCacheManager> dentryManager,
+                   std::shared_ptr<InodeCacheManager> inodeManager,
+                   std::shared_ptr<MdsClient> mdsClient);
+
+    CURVEFS_ERROR GetTxId();
+    CURVEFS_ERROR Precheck();
+    CURVEFS_ERROR PrepareTx();
+    CURVEFS_ERROR CommitTx();
+    void DeleteOldInode();
+    void UpdateCache();
+
+ private:
+    CURVEFS_ERROR CheckOverwrite();
+
+ private:
+    uint32_t fsId_;
+    uint64_t parentId_;
+    std::string name_;
+    uint64_t newParentId_;
+    std::string newname_;
+
+    uint32_t srcPartitionId_;
```

Comment:
those feild should initialize a default value for defensive programming

Reply:
> Those members should initialize a default value in constructor for defensive programming

yep, i will improve it.